nucleus/deploy/client.py

```diff
@@ -103,6 +107,7 @@ def create_model_bundle(
             model: Typically a trained Neural Network, e.g. a Pytorch module
             load_model_fn: Function that when run, loads a model, e.g. a Pytorch module
             load_predict_fn: Function that when called with model, returns a function that carries out inference
```

Comment:
It looks like the original type annotation of `load_predict_fn` was a `Callable`: `Callable[[DeployModel_T], Callable[[Any], Any]]`. In that case, maybe we don't need to create a new `predict_fn_or_cls` field, since `load_predict_fn` was already forwards compatible with the way you've specified the Bundle class with a `__call__` method?

Reply:
They have different usage:
a. predict_fn_or_cls
```
foo("woo") # returns inference result;
client.upload_bundle(predict_fn_or_cls=foo)
```

b. load_predict_fn
```
foo("woo") # returns inference result;

# incorrect:
client.upload_bundle(load_predict_fn=foo)

# correct:
def load_predict_fn(model):
    def predict(args):
          return foo(args)
    return predict
```

Reply:
I mean `predict_fn_or_cls` is a function/class that you can use directly in the service. `load_predict_fn ` on the other hand is a factory that produces a callable object.

Reply:
Oh I see, the absence or presence of the `load` prefix can serve as a distinction as to whether or not it's directly used or not 👌🏻